### ISSUE TITLE
cannot reproduce UCR result

### ISSUE THREAD

**Issue by fire717** [Alrash/TimesURL#7]
Hi,
I am working on times series project and try to use TimesURL,
while u said it got Avg.ACC 0.845 on 128UCR datasets,
I can get only 0.7857.

I just clone this project and download the UCR dataset, 
And I found the train script only support one folder,
So I add a list to save each result then run mean to get avg acc.

Did I miss anything? Thx!

(I found a UCR_Missing_value_and_variable_length_datasets_adjusted in UCR and I didnot use it, is it the key and should I use it?)

**Comment by Alrash** [comment]
Thank you for your interest in our research. Please note that for the UCR dataset, the hyperparameters used to achieve the reported results differ from the default settings in our training script. For instance, we use a batch size of 32, a learning rate of 1e-5, and a seed of 42. As a result, reproducing the paper's results may require some hyperparameter tuning.
